tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,92 @@
+    # Click Element                   //div[@data-aura-rendered-by='106:0']//input[@type='radio']
+    # Capture Page Screenshot
+
+    #Go into Edit mode 
+    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
+    Wait Until Element Is visible   //div[@data-aura-rendered-by='28:0']
+    Capture Page Screenshot
+
+
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'Academic Program')]
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+    # Unselect Checkbox               //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Academic Program')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Academic Program')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+
+    Capture Page Screenshot
+
+    Sleep                           10
```

Comment:
Is the sleep still necessary here now that we wait for all requests to finish processing?

Reply:
Not needed so I removed it.